src/SourceMapToolkit.CallstackDeminifier/StackTraceDeminifier.cs

```diff
@@ -37,7 +37,26 @@ public DeminifyStackTraceResult DeminifyStackTrace(string stackTraceString, bool
 			}
 
 			deminifiedStackFrameResults.Reverse();
-
+			if (fixOffByOne)

```

Comment:
I assume this only makes sense when `preferSourceMapsSymbols == true`? Should this condition then be `if (preferSourceMapsSymbols && fixOffByOne)`?

Reply:
agree